docs/product/accounts/migration/index.mdx

```diff
@@ -16,65 +16,72 @@ For additional reading on considering SaaS, take a look at:
 
-Take a look at our [plans](https://sentry.io/pricing/) and select one that suits your needs based on the different features and capabilities. Note that plan pricing varies based on the type of _Contract_ and _Monthly Event Volume_, so make sure you are familiar with your current/expected monthly usage (number of events per month).
+## How to Relocate
 
-If you're expecting higher volumes or you're interested in our Enterprise capabilities (priority support and SLAs, and single-tenant options, for example), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
+### 1. Sign up with a promo code
 
-### 3. Export your data
+Before you begin the relocation process, [sign up](https://sentry.io/signup/relocate/) for a new account on Sentry. If you already have a Sentry account, login and navigate to the [relocation landing page](https://sentry.sentry.io/relocation/). In either case, you should see a page like this:
 
-Self-hosted Sentry provides a script that allows you to perform a backup of your basic data into a json file. For versions of self-hosted under 23.3.1, an `export` command is required instead. See [here](https://develop.sentry.dev/self-hosted/backup/) for more.
+![Relocation getting started page](./img/relocation-landing-page.png)
 
-Run the following command in your terminal to start the backup script and export your data:
+**To receive a $100 credit, make sure to use the promo code `relocate100` on this page.**
 
-    $ ./scripts/backup.sh
+### 2. Choose which organizations to relocate
 
-This invokes the sentry CLI `export` command on a new isolated instance of our Docker web container.
+Your self-hosted installation may have had multiple organizations, each identified by a unique slug. You'll need to collect those from your installed instance, and decide which ones you'll want to keep. You can always change these to whatever you like post-relocation on the organization settings page.
 
-The export migrates your account's settings and configurations, including:
+![Changing the org slug on the settings page](./img/org-slug.png)
 
-- Project settings
-- Alert rules
-- Enabled data scrubbers
-- Team settings
-- Saved search queries
-- Integration platform metadata (Sentry 10)
+### 3. Follow the relocation instructions
 
-### 4. Send it to us
+The remaining pages in the relocation instructions will contain specific details on how to export your self-hosted instance's relocatable data. This will require terminal access to the machine running your self-hosted instance. In brief, you'll save a public key that we generate for you on your self-hosted machine, which you'll then use to execute a command that we provide to generate an encrypted tarball containing your data. You'll then upload this tarball, and we'll take it from there!
```

Comment:
```suggestion
The rest of the in-app relocation instructions will walk you through the specific details of how to export your self-hosted instance's relocatable data. This will require terminal access to the machine running your self-hosted instance. Here's a quick overview:

- You'll be asked to save a public key (that we generate on your self-hosted machine)
- You'll then use this key to execute a command which will generate an encrypted tarball with all your data
-Lastly, you'll be prompted to upload this tarball 
```